Packs/Zoom/Integrations/Zoom/Zoom.py

```diff
@@ -1624,43 +2084,366 @@ def zoom_list_messages_command(client, **args) -> CommandResults:
+        url_suffix = '/im/chat/messages'
+        json_data_all = {
+            "content": {
+                "body": [
+                    {
+                        "type": "message",
+                        "text": message
+                    }
+                ]
+            },
+            "to_jid": channel_jid,
+            "robot_jid": client.botJid,
+            "account_id": client.account_id
+        }
+        client.zoom_send_notification(url_suffix, json_data_all)
+    # if kick_admin:
+    #     demisto.debug("kick-admin:")
+    #     url_suffix = f'/chat/users/{admin_user_id}/channels/{channel_id}/members/{admin_user_id}'
+    #     res = client.zoom_remove_from_channel(url_suffix)
+    #     demisto.debug(f"res: {res}")
+    return CommandResults(
+        readable_output=f'Investigation mirrored successfully, channel:{channel_name},{channel_jid}'
+    )
+
+
+def find_mirror_by_investigation() -> dict:
+    """
+    Finds a mirrored channel by the mirrored investigation
+
+    Returns:
+        The mirror object
+    """
+    mirror: dict = {}
+    investigation = demisto.investigation()
+    demisto.debug(f"investigation: {investigation}")
+    if investigation:
+        integration_context = get_integration_context(SYNC_CONTEXT)
+        demisto.debug(f"integration_context: {integration_context}")
+        if integration_context.get('mirrors'):
+            mirrors = json.loads(integration_context['mirrors'])
+            investigation_filter = list(filter(lambda m: investigation.get('id') == m['investigation_id'],
+                                               mirrors))
+            if investigation_filter:
+                mirror = investigation_filter[0]
+
+    return mirror
+
+
+def close_channel(client, **args):
+    """
+        if AutoClose is true in mirroring
+        delete the mirrored zoom channel from zoom and from the context
+    """
+    demisto.debug(f"args close channel: {args}")
+    client = client
+    admin_user_id = get_user_id_from_token(client)
+    mirror = find_mirror_by_investigation()
+    demisto.debug(f"mirror: {mirror}")
+    channel_id = None
+    integration_context = get_integration_context(SYNC_CONTEXT)
+    if mirror:
+        mirrors = json.loads(integration_context['mirrors'])
+        channel_id = mirror.get('channel_id', '')
+        channel_mirrors = list(filter(lambda m: channel_id == m['channel_id'], mirrors))
+        demisto.debug(f"channel: {channel_id}")
+        for mirror in channel_mirrors:
+            mirror['remove'] = True
+            demisto.mirrorInvestigation(mirror['investigation_id'], f'none:{mirror["mirror_direction"]}',
+                                        bool(mirror['auto_close']))
+        set_to_integration_context_with_retries({'mirrors': mirrors}, OBJECTS_TO_KEYS, SYNC_CONTEXT)
+    if channel_id:
+        zoom_delete_channel_command(client=client, channel_id=channel_id, user_id=admin_user_id)
+        return 'Channel successfully delete.'
```

Comment:
In any case, @ShirleyDenkberg should go over the English text.
```suggestion
        return f'Channel {channel_id} was successfully deleted.'
```